scripts-available/CDB_Organizations.sql

```diff
@@ -25,6 +25,57 @@ BEGIN
 END
 $$ LANGUAGE PLPGSQL VOLATILE;
 
+-------------------------------------------------------------------------------
+-- Administrator
+-------------------------------------------------------------------------------
+CREATE OR REPLACE
+FUNCTION cartodb._CDB_Organization_Admin_Role_Name()
+    RETURNS TEXT
+AS 'SELECT ''cdb_org_admin''::text || ''_'' || md5(current_database());'
```

Comment:
Ok, changed to `'SELECT current_database() || ''_admin''::text;'` removing md5 and making naming more consistent by prepending the database. Since its length doesn't depend on user data and database_length is under control in CartoDB I think we don't need to add checks. We've successfully generated larger `cdb_org_member` role so far indeed.